doc/howto/write_an_alphabet/index.md

```diff
@@ -35,9 +37,13 @@ As a consequence, we denote Guanine and Cytosine as strong (S) and Adenine and T
 In this section we want to implement a seqan3::Alphabet that consists of the characters `S` and `W` to represent
 strong and weak nucleobases.
 
-Let's start with a simple struct that only holds the alphabet's numerical representation, namely the **rank** value:
+Let's start with a simple struct that only holds the alphabet's numerical representation, namely the **rank** value.
 \snippet dna2_only_rank.cpp struct
```

Comment:
"It is not specified how the value of an alphabet is stored internally, however alphabets typically store the rank in as a member variable."